pkg/transform/kubernetes/kubernetes.go

```diff
@@ -77,6 +77,67 @@ func (t *Transformer) CreateServices(o *object.Service) ([]runtime.Object, error
 	return result, nil
 }
 
+// Create k8s ingresses for OpenCompose service
+func (t *Transformer) CreateIngresses(o *object.Service) ([]runtime.Object, error) {
+	result := []runtime.Object{}
+
+	i := &ext_v1beta1.Ingress{
+		ObjectMeta: api_v1.ObjectMeta{
```

Comment:
Not really sure what you would like to get abstracted here. The object data are always different.